app/views/patients/_patient_information.html.erb

```diff
@@ -42,7 +42,7 @@
         </div>
 
         <%= f.text_field :county, autocomplete: 'off' %>
-        <%= f.number_field :zipcode, autocomplete: 'off' %>
+        <%= f.text_field :zipcode, autocomplete: 'off', pattern: '\d{5}'%>
```

Comment:
I suggest we support five digit, nine digit, and 5-4 digits!
https://faq.usps.com/s/article/ZIP-Code-The-Basics

Reply:
Now with more ZIP!

Supports normal (12345) and ZIP+4 (12345-6789). Nine digit (123456789) will be converted to ZIP+4 (dash added). Do we need to support any other separators? USPS says no, but I guess there's a question of whether we want to allow spaces, periods, etc.